yarn-project/p2p/src/services/reqresp/reqresp.ts

```diff
@@ -439,6 +439,12 @@ export class ReqResp {
     } catch (e: any) {
       this.metrics.recordRequestError(subProtocol);
       this.handleResponseError(e, peerId, subProtocol);
+
```

Comment:
Update this method such that it never returns undefined, but a explicit status.FAILURE